File: ChatSupport/src/main/java/com/vantagecircle/chatapp/core/interfaceC/ValueInterface.java
package com.vantagecircle.chatapp.core.interfaceC;

import com.vantagecircle.chatapp.core.model.DataModel;

/**
 * Created by bapidas on 01/08/17.
 */

public interface ValueInterface{
    void onDataSuccess(DataModel dataModel);
    void onDataCancelled(DataModel dataModel);
}

File: app/src/main/java/com/bapidas/chattingapp/data/core/callbacks/ValueInterface.kt
package com.bapidas.chattingapp.data.core.callbacks

import com.bapidas.chattingapp.data.core.model.DataModel

/**
 * Created by bapidas on 01/08/17.
 */
interface ValueInterface {
    fun onDataSuccess(dataModel: DataModel)
    fun onDataCancelled(dataModel: DataModel)
}